docs/secure-connections/sauce-connect/setup-configuration/docker.md

```diff
@@ -123,6 +155,72 @@ $ cat /tmp/sc.ready | jq .
 }
 ```
 
+## Running the Sauce Connect Docker in the nginx Server
+
+The `docker-compose.yaml` file below shows how to run Sauce Connect Docker in the nginx server.
+
+The Sauce Connect Proxy sits adjacent to the `nginx:latest` instance: you can reach the nginx service from Sauce Labs by going to nginx (if you run it as is). You can also provide a name to the container with [container_name](https://docs.docker.com/compose/compose-file/compose-file-v3/#container_name).
+
+`container_name: mockserver` is reachable from any app or website that tried to hit mockserver via HTTP: this is due to the nature of [how docker resolves container names](https://docs.docker.com/config/containers/container-networking/#:~:text=In%20the%20same,on%20that%20network) for any valid entries on that docker network.
```

Comment:
```suggestion
If you were to name the container explicitly like `container_name: mywebserver` it would reachable from any app or website that tried to hit mockserver via HTTP: this is due to the nature of [how docker resolves container names](https://docs.docker.com/config/containers/container-networking/#:~:text=In%20the%20same,on%20that%20network) for any valid entries on that docker network.
```